Felienne/hedy
# Enable the turtle commands in all levels

**Enable turtle commands for all levels**

_Deletes the explicit removal of turtle commands in level 11._
_Also updates the tests constant for the turtle level so it continues after level 10._

**Fixes #3299**

**How to test**

Go to any level 11 - 18 and verify that the turtle commands work and produce the expected outcome.
Example code:
```
forward 50
color green
turn 45
forward 50
```
